ramls/userEvent.json

```diff
@@ -38,6 +38,10 @@
       "description": "Full snapshot of the user",
       "type": "object",
       "$ref": "userdata.json"
+    },
+    "personalDataChanged": {
```

Comment:
Is it ok that we have in json 2 name conventions: isPersonalDataChanged and personalDataChanged ?